pkg/unvanquished_src.dpkdir/bots/default.bt

```diff
@@ -4,11 +4,25 @@ selectClass
 	{
 		condition team == TEAM_ALIENS
 		{
-			spawnAs PCL_ALIEN_LEVEL0
+			selector
+			{
+				condition levelTime >= 60000 && ( numUsersInTeam == 0 || numOurBuildings( E_A_SPAWN ) < 2 ) && usableBuildPoints >= chosenBuildableCost && !teamateHasWeapon( WP_ABUILD ) && !teamateHasWeapon( WP_ABUILD2 )
+				{
+					spawnAs PCL_ALIEN_BUILDER0
+				}
+				spawnAs PCL_ALIEN_LEVEL0
+			}
 		}
 		condition team == TEAM_HUMANS
 		{
-			spawnAs WP_MACHINEGUN
+			selector
+			{
+				condition levelTime >= 60000 && ( numUsersInTeam == 0 || numOurBuildings( E_H_SPAWN ) < 2 ) && usableBuildPoints >= chosenBuildableCost && !teamateHasWeapon( WP_HBUILD )
```

Comment:
So that the num spawns cvars work correctly I guess `numOurBuildings( E_H_SPAWN ) < 2` should be replaced with `(numOurBuildings( E_H_SPAWN ) < 2 && numOurBuildings( E_H_SPAWN ) < cvar( "g_bot_buildNumTelenodes" )` ?

P.S. you can split the condition on multiple lines. No magic whitespace in BT